### PR TITLE
Update deployment.toml configs for Berlin Toolkit

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to update the docs with the deployment.toml configs for Berlin toolkit.

### CODE REVIEW

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md [hunk 1]
```diff
@@ -123,12 +123,29 @@ database server, and the JDBC driver.
...
+
+    ``` toml
+    [open_banking_berlin.consent]
+    supported_challenge_methods = ["S256"]
+    ```
+    
+14. To generate the self link in the consent JSON response, configure the URLs of the exposed APIs as follows:
    
     ``` toml
     [open_banking_uk.consent]
```
Comment (aka4rKO) [inline en/docs/install-and-setup/configuring-identity-server-for-ob.md]: this config is not part of berlin

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md [hunk 2]
```diff
@@ -137,23 +154,23 @@ the consent page.
     cof_consent_self_link = "https://<APIM_HOST>:8243/open-banking/{version}/cbpii/"
     ```
 
-12. In the consent re-authentication step of the Accounts flow, during authorisation, the PSU is allowed to change the 
+15. In the consent re-authentication step of the Accounts flow, during authorisation, the PSU is allowed to change the 
 selected account. To enable this feature and update the account bound to the consent, set the following property to true:
 
     ``` toml
     [open_banking_uk.consent]
```
Comment (aka4rKO) [inline en/docs/install-and-setup/configuring-identity-server-for-ob.md]: this config is not part of berlin

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md [hunk 3]
```diff
@@ -137,23 +154,23 @@ the consent page.
...
     acc_update_by_psu_enabled = true
     ```
 
-13. Enable Request-URI validation that validates `AccountID` in the request against the `AccountID` in consent during 
+16. Enable Request-URI validation that validates `AccountID` in the request against the `AccountID` in consent during 
 account retrieval. By default, this is disabled and the configuration is set to `false`.
 
     ``` toml
     [open_banking_uk.consent]
```
Comment (aka4rKO) [inline en/docs/install-and-setup/configuring-identity-server-for-ob.md]: not related to berlin

**Comment by isharailanga** [comment]
Added changes according to  [Berlin Toolkit - Configuring API Manager](https://docs.google.com/document/d/1Fzo0h3MChs4KZZrv1ebKLVA7nsjOZTgG2JwGdfIQmqE/edit) via 2325df41d882afbbc9a1e9cd1235295d598a2189 & 690cc302c7aeebd8f52017e85fc4372c2c6bdccd.

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 4]
```diff
@@ -101,7 +84,7 @@ the consent page.
     sharable_account_retrieval_endpoint = "http://<APIM_HOST>:9763/api/openbanking/uk/backend/services/bankaccounts/bankaccountservice/sharable-accounts"
     ```
    
-9. To generate the self link in the consent JSON response, configure the URLs of the exposed APIs as follows:
+8. To generate the self link in the consent JSON response, configure the URLs of the exposed APIs as follows:
 
     ``` toml
     [open_banking_uk.consent]
```
Comment (isharailanga) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md]: @aka4rKO are these tags required in the IS toml?

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 5]
```diff
@@ -110,40 +93,60 @@ the consent page.
     cof_consent_self_link = "https://<APIM_HOST>:8243/open-banking/{version}/cbpii/"    
     ```
     
-10. Enable Request-URI validation that validates `AccountID` in the request against the `AccountID` in consent during 
+9. Enable Request-URI validation that validates `AccountID` in the request against the `AccountID` in consent during 
 account retrieval. By default, this is disabled and the configuration is set to `false`.
 
     ``` toml
     [open_banking_uk.consent]
```
Comment (isharailanga) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md]: @aka4rKO are these tags required in the IS toml?

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-identity-server-for-ob.md]: Fixed in 95d4998d1408c1aa79d8e534e6b4cabe97d99a68

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-identity-server-for-ob.md]: Fixed in 95d4998d1408c1aa79d8e534e6b4cabe97d99a68

File: en/docs/install-and-setup/configuring-identity-server-for-ob.md (hunk 3 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-identity-server-for-ob.md]: Fixed in 95d4998d1408c1aa79d8e534e6b4cabe97d99a68

**Comment by isharailanga** [comment]
@aka4rKO @Akila94  please verify the tags on https://docs.google.com/document/d/17_QebQ-cfeKNbPqcTVnhLmwtY-sHW32YRks2hJD1NJA/edit# as well.

**Review by SachiniSiriwardene — APPROVED** [review]

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md]: Removed

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 5 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md]: Updated with the correct config

**Pull request merged**